index.html

```diff
@@ -1,76 +1,241 @@
+            <nav class="navbar">
+                <ul class="nav-links">
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#examples">Examples</a></li>
+                    <li><a href="https://github.com/jcubic/chat-gpt" target="_blank" rel="noopener">GitHub</a></li>
+                </ul>
+            </nav>
+        </div>
+        <img src="assets/down.png" alt="Scroll down" class="scroll-indicator" />
+    </header>
+
+    <main>
+        <section id="about">
+            <div class="container">
+                <h2>What is this?</h2>
+                <p>
+                    This is a small tool that lets you save ChatGPT conversations to your computer
+                    with a single click, using a JavaScript bookmarklet.
+                </p>
+                <p>
+                    Originally introduced in a <a href="https://dev.to/jcubic/save-chatgpt-as-html-file-dhh">DEV.to
+                        article</a>,
+                    it is now maintained on GitHub to ensure continued compatibility with changes in OpenAI's UI.
+                </p>
+            </div>
+        </section>
+
+        <section>
+            <div class="container">
+                <h2>Why does this exist?</h2>
+                <p>
+                    Since ChatGPT’s interface can change at any time, this tool uses a CDN-hosted script from GitHub.
+                    This keeps it working even when OpenAI updates their web UI.
+                </p>
+            </div>
+        </section>
+
+        <section>
+            <div class="container">
+                <h2>How do I use it?</h2>
+                <p>
+                    To run the code you need to create a new bookmark and copy the code from the JavaScript file.
+                </p>
+                <div class="button-like">
+                    <button class="copy-button" onclick="copyBookmarklet()">Copy Bookmarklet</button>
+                  </div>
+                  <textarea id="bookmarklet-code" readonly style="display: none; margin-top: 1rem; width: 100%; max-width: 700px; height: 120px;"></textarea>
+                                    
+
+                <p>
+                    If you don't know how, you can drag & drop the
+                    <strong><a id="bookmark" href="#">Save ChatGPT</a></strong> link to your bookmarks.
+                </p>
+            </div>
+        </section>
+
+        <section id="examples">
+            <div class="container">
+                <h2>Examples</h2>
+                <p>See how others used the bookmarklet:</p>
+                <ul>
+                    <li><a href="./2025-04-11/chat-gpt-scan-lan-for-fedora.html">Scan LAN for Fedora</a> (2025-04-11)
+                    </li>
+                    <li><a href="./2025-03-16/chat-gpt-clear-ansi-term-emacs.html">Clear ansi-term Emacs</a>
+                        (2025-03-16)
+                    </li>
```

Comment:
Last thing. Please make the html consistent, always keep `<li>` and every thing inside in one line. or use something like this:
```html
<li>
   <a href="./2025-04-11/chat-gpt-scan-lan-for-fedora.html">Scan LAN for Fedora</a> (2025-04-11)
</li>
```

Right now, the HTML formatting is random. You can use some pretty print tool or service if you want.